resources/views/spa.blade.php

```diff
@@ -24,13 +26,31 @@
 	<meta name="theme-color" content="#ffffff">
 	<meta name="msapplication-config" content="favicon/browserconfig.xml" />
 
+	<script>
+    var m_id = '<?php echo $measurement_id; ?>';
+    var src_url = 'https://www.googletagmanager.com/gtag/js?id=' + m_id; // Appending m_id to URL
+	//alert(src_url);
+	</script>
+
+	<!-- Global site tag (gtag.js) - Google Analytics -->
+	<?php 
+	if($measurement_id != 'None') {
```

Comment:
it is always suggested to use `!==` than `!=` as php like javascript does not check its type if you just use 1 equal